galera/master.sls

```diff
@@ -21,123 +21,112 @@ xtrabackup_repo_fix:
       - pkg: galera_packages
 {%- endif %}
 
-galera_packages:
-  pkg.installed:
-  - names: {{ master.pkgs }}
-  - refresh: true
-
-galera_log_dir:
-  file.directory:
-  - name: /var/log/mysql
-  - makedirs: true
-  - mode: 755
-  - require:
-    - pkg: galera_packages
 
 {%- if grains.os_family == 'Debian' %}
-galera_run_dir:
-  file.directory:
-  - name: /var/run/mysqld
-  - makedirs: true
-  - mode: 755
-  - user: mysql
-  - group: root
-  - require:
-    - pkg: galera_packages
-{%- endif %}
-
-galera_init_script:
+mariadb_repo:
   file.managed:
-  - name: /etc/init.d/mysql
-  - source: salt://galera/files/mysql
-  - mode: 755
-  - require: 
-    - pkg: galera_packages
+  - name: /etc/apt/sources.list.d/mariadb_10-1.list
+  - source: salt://galera/files/mariadb.list
 
-galera_bootstrap_script:
+mariadb_key:
```

Comment:
Don't setup repositories in formulas, it was agreed on one of openstack-salt meetings that best practice is to keep repository setup up to other formulas (linux) or components (eg. setup everything in image, or cloud-init, whatever) to satisfy most use-cases (eg. using mirrors, custom repositories, etc.)